<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>448</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="MainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="HeadWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="LOG">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image:url(:/image/Ico/Catlog1.ico)</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_top">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>646</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_button">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="MinmizeButton">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{   
    border-image: url(:/image/Ico/min_gray.ico);
    background:transparent;
}
QPushButton:hover
{
    border-image: url(:/image/Ico/min_white.ico);
    background:transparent; 
}

QPushButton:pressed
{
    border-image: url(:/image/Ico/min_gray.ico);
	background:transparent; 
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="CloseButtion">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{   
    border-image: url(:/image/Ico/close_gray.ico);
    background:transparent;
}
QPushButton:hover
{
    border-image: url(:/image/Ico/close_red.ico);
    background:transparent; 
}

QPushButton:pressed
{
    border-image: url(:/image/Ico/close_gray.ico);
    background:transparent;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="LeftWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_button">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="TestButton">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{   
    border-image: url(:/image/Ico/test_gray.ico);
    background:transparent; 
}
QPushButton:hover
{
    border-image: url(:/image/Ico/test_white.ico);
	background:transparent; 
}

QPushButton:checked
{
    border-image: url(:/image/Ico/test_white.ico);
	background:transparent; 
	/* 边框风格 */  
    border-style:outset;  
    /* 边框宽度 */  
    border-width:2px;  
    /* 边框颜色 */  
    border-color:#1C1C1C;  
}
QToolTip{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
/*
QPushButton::toolTip
{
	background-color: rgb(36, 36, 36);
}*/
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="SettiongButton">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{   
    border-image: url(:/image/Ico/setting_gray.ico);
    background:transparent; 
}
QPushButton:hover
{
    border-image: url(:/image/Ico/setting_white.ico);
	background:transparent; 
}

QPushButton:checked
{
    border-image: url(:/image/Ico/setting_white.ico);
	background:transparent; 
	/* 边框风格 */  
    border-style:outset;  
    /* 边框宽度 */  
    border-width:2px;  
    /* 边框颜色 */  
    border-color:#1C1C1C;  
}
QToolTip{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="LogButton">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{   
    border-image: url(:/image/Ico/log_gray.ico);
    background:transparent; 
}
QPushButton:hover
{
    border-image: url(:/image/Ico/log_white.ico);
	background:transparent; 
}

QPushButton:checked
{
    border-image: url(:/image/Ico/log_white.ico);
	background:transparent; 
	/* 边框风格 */  
    border-style:outset;  
    /* 边框宽度 */  
    border-width:2px;  
    /* 边框颜色 */  
    border-color:#1C1C1C;  
}
QToolTip{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_left">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>269</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="HelpButton">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{   
    border-image: url(:/image/Ico/help_gray.ico);
    background:transparent; 
}
QPushButton:hover
{
    border-image: url(:/image/Ico/help_white.ico);
	background:transparent; 
}

QPushButton:pressed
{
    border-image: url(:/image/Ico/help_gray.ico);
	background:transparent;  
}
QToolTip{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="FunctionWidget">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="Test"/>
          <widget class="QWidget" name="Setting"/>
          <widget class="QWidget" name="Log"/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="BottomWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="Hintlabel">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>752</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QProgressBar" name="ProgressBar">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>10</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="tabletTracking">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar {   
	border: 1px solid grey;   
	border-radius: 5px;   
	background-color: #FFFFFF;
}
QProgressBar::chunk {   
	background-color: #92bd6c;   
	width: 20px;
}
QProgressBar {   
	border: 1px solid grey;   
	border-radius: 5px;   
	text-align: center;
}</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
